src/store/login/login.sagas.js

```diff
@@ -1,8 +1,11 @@
+/* eslint-disable func-names */
+/* eslint-disable no-console */
 import { call, takeLatest, put } from 'redux-saga/effects';
 import jwt from 'jwt-decode';
 import { LOGIN_ACTIONS, LOGIN_ACTION_CREATORS } from './login.actions';
 import { USER_ACTION_CREATORS } from '../user/user.actions';
```

Comment:
I really like how you're using these three object here instead of importing all of their functions. Even though there are pros and cons to this approach, I think that it actually makes it more readble